SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -280,14 +293,15 @@ private ScriptRunner()
             _triggerHandler = new Dictionary<Trigger, EventHandler<Tuple<Datum, Datum>>>();
             _maximumAgeMinutes = 10;
             _randomTriggerWindows = new List<Tuple<DateTime, DateTime>>();
-            _randomTriggerCallbackId = null;
+            _randomTriggerWindowCallbacks = new SortedDictionary<string, Tuple<DateTime, DateTime>>();
```

Comment:
No good reason anymore- planned to replace `_randomTriggerWindows` (which is sorted) with the dictionary but turned out to be simpler not to. I'll switch it. Though maybe we should use `ConcurrentDictionary` since it's thread safe? I don't immediately see a case where `SensusServiceHelper` and `ScriptRunner` would access the dictionary at the same time and cause problems, but both do have access to it.